plugins/filter_kubernetes/kubernetes.c

```diff
@@ -729,6 +729,16 @@ static struct flb_config_map config_map[] = {
      "Kubernetes authorization token file"
     },
 
+#ifdef FLB_HAVE_KUBE_TOKEN_COMMAND
+    /* Kubernetes Token command */
+    {
+     FLB_CONFIG_MAP_STR, "kube_token_command", NULL,
+     0, FLB_FALSE, 0,
```

Comment:
why not storing the value ?

Reply:
Because we have black spaces in the command and I tried to store the value by `offset`. It would fail to get the full command

Reply:
Yeah we use `flb_filter_get_property` because that seemed to work nicely...